backend/routes/api.php

```diff
@@ -33,4 +33,6 @@
     Route::apiResource('auth', AuthController::class)->only('update')->parameters(['auth' => 'user']);
     Route::apiResource('users.accounts', AccountController::class)->shallow()->only(['index']);
     Route::apiResource('accounts', AccountController::class)->only(['index', 'show']);
+
+    Route::get('search', [UserController::class, 'search']);
```

Comment:
will the resulting route be just `/search`?

since this is search for users, can we have it something like this
`/users/search`